modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -326,9 +326,9 @@ The registration token includes your organization's ID and the current environme
 . Click *Copy* to copy the command.
 . In a terminal window, change to the `$MULE_HOME/bin` directory for the Mule instance that you're registering.
 . Paste the command on the command line.
-. Change the instance name `server-name` to the unique name to use to label Mule in the Runtime Manager console.
 +
-The `server-name` value is unique across environments. The `server-name` can be the same for the same organization in different environments.
+The command includes the unique `server-name` used to label Mule in the Runtime Manager console.
```

Comment:
Asking with a newbie hat on: Does the customer explicitly add the unique server-name when executing the command? Or does the command already include the server-name? Or perhaps this is explained in a previous part of the document?

Reply:
The customer names the server in RTM and the command includes the server-name the customer-defined. I would have removed this entirely, but I remember adding it in to clarify at the request of someone a while back.